src/absil/illib.fs

```diff
@@ -385,6 +385,11 @@ module List =
         mn 0 xs
     let count pred xs = List.fold (fun n x -> if pred x then n+1 else n) 0 xs
 
+    let headAndTail l = 
```

Comment:
This was intentionally removed by @forki in another PR and he cleaned up (and simplified) some places where it was being used. Is it necessary to add back here?

Reply:
I moved Don's cheese ;-)

Reply:
Yeah this is a useful function :-)